llvm/test/Transforms/OpenMP/hide_mem_transfer_latency.ll

```diff
@@ -1,5 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: -p --function-signature --scrub-attributes
-; RUN: opt -S -openmp-opt-cgscc -aa-pipeline=basic-aa -openmp-hide-memory-transfer-latency < %s | FileCheck %s
+; Pass is not designed for the old pass manager -- CallGraph is not updated.
```

Comment:
IIRC, @AlexeySachkov, created an issue about enabling new PM. Can you add a link to the issue here and in llvm/test/Transforms/OpenMP/values_in_offload_arrays.ll test, please?

Reply:
@bader I can't find this issue, unfortunately. 
I hope these comments will help to fix test failures once we enable the new PM.

Reply:
@AlexeySachkov, how do we track changes we did to llorg sources to disable NewPM?

Reply:
> IIRC, @AlexeySachkov, created an issue about enabling new PM.

I didn't create such issue